openfl/text/TextField.hx

```diff
@@ -2082,13 +2106,9 @@ class TextField extends InteractiveObject {
 		switch (key) {
 			
 			case RETURN, NUMPAD_ENTER:
-				
-				if (__textEngine.multiline) {
-					
-					replaceSelectedText ("\n");
-					dispatchEvent (new Event (Event.CHANGE, true));
-					
-				}
```

Comment:
Looking back at it, I think it was a result of a merge that removed the if condition. It seems that the if condition was added a few days prior to this PR in 2dd1c15c0b8abb98905ff5be2d8ed09f1acda875. I don't see any reason why I would have removed it intentionally.